packages/client-core/src/components/MediaIconsBox/index.tsx

```diff
@@ -53,15 +53,22 @@ import Icon from '@etherealengine/ui/src/primitives/mui/Icon'
 import IconButtonWithTooltip from '@etherealengine/ui/src/primitives/mui/IconButtonWithTooltip'
 
 import { FeatureFlags } from '@etherealengine/common/src/constants/FeatureFlags'
+import multiLogger from '@etherealengine/common/src/logger'
 import useFeatureFlags from '@etherealengine/engine/src/useFeatureFlags'
 import { isMobile } from '@etherealengine/spatial/src/common/functions/isMobile'
 import { VrIcon } from '../../common/components/Icons/VrIcon'
 import { SearchParamState } from '../../common/services/RouterService'
 import { RecordingUIState } from '../../systems/ui/RecordingsWidgetUI'
 import { MediaStreamService, MediaStreamState } from '../../transports/MediaStreams'
+import { clientContextParams } from '../../util/contextParams'
 import { useShelfStyles } from '../Shelves/useShelfStyles'
 import styles from './index.module.scss'
 
+const clogger = multiLogger.child({
```

Comment:
My apologies for the late response, some I missed these comments until I wanted to verify why the story isn't in done already.

Anyway, I made most of the change requests excepting this one, the reason I created a second logger instance is because the previously existing one is being used in a bunch of logs that are completely unrelated to the events we want to track for Analytics.

As I have mentioned, the reason I decided not to globally add the context params to every log in favor of the optional modifier hook is because it can be enabled only for the logs requiring to have these params for the sake of avoiding any negative impact into the previous implementations of the multilogger utility, perfect example of what I mean is the issue brought up by Josh mentioning that it became difficult to debug logs with properties having nothing to do with Analytics related logs.

I'm open for other naming suggestions rather than "clogger", but I'm not keen about reusing the previously existing "logger" instance because by doing that it defeats the purpose which is avoiding negative impact everywhere else the multilogger utility is being used.

Cheers,
Jon

Reply:
The thing is `import logger from '@etherealengine/common/src/logger'` is wrong import and there would be no information regarding which `component:` the log is coming from.

Reply:
Yeah that makes sense, I just verified and is not creating an instance using multilogger.child I see what you mean, though that is existing code, it isn't introduced in this PR, since it's being used....
![image](https://github.com/user-attachments/assets/1728fc0b-2889-4d90-8bed-46851822f117)

I guess the question is, should I remove the import line and where it's being used, or to simply remove the bad import and use the multilogger reference I did import to create the `logger = multilogger.child({ component: '' })` having no contextParams?

I'm usually not keen to introduce changes in PRs that are unrelated to the assignment, but this is a quick change so I have no problem making an exception.

Reply:
I would suggest `remove the bad import and use the multilogger reference`.

Thanks.